src/fmt/writer/termcolor/extern_impl.rs

```diff
@@ -452,6 +452,7 @@ impl_styled_value_fmt!(
 /// Hexadecimal numbers are written with a `0x` prefix.
 #[allow(missing_docs)]
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
```

Comment:
The same as the manual implementation.

Reply:
This would bump the MSRV from currently 1.31 to 1.40.